src/czar/MessageTable.cc

```diff
@@ -138,7 +138,8 @@ void MessageTable::_saveQueryMessages(ccontrol::UserQuery::Ptr const& userQuery)
     cMsg += multiErrStr;
     LOGS(_log, LOG_LVL_DEBUG, " MULTIERROR:" << cMsg);
     std::string summaryQ = (boost::format(::writeTmpl) % _tableName % "-1" % "-1" %
-                            _sqlConn->escapeString(cMsg) % severity % std::time(nullptr)).str();
+                            _sqlConn->escapeString(cMsg) % severity % std::time(nullptr))
+                                   .str();
```

Comment:
Putting this to the next line doesn't look right. Did you do this, or this was done automatically by the `clang-format` tool?

Reply:
That's the formatter, not me.